docs/en/swip/readme.md

```diff
@@ -72,6 +72,7 @@ Next SWIP Number: 6
 
 ### Accepted SWIPs
 
+- [SWIP-6 Support ActiveMQ Monitoring](SWIP-6.md)
```

Comment:
Could you update `Next SWIP Number: 6` to `Next SWIP Number: 8`? 7 was declared by https://github.com/apache/skywalking/discussions/12071, although it is not done yet.

Reply:
OK